src/module.c

```diff
@@ -75,6 +79,26 @@ static int parseCreateArgs(RedisModuleCtx *ctx, RedisModuleString **argv, int ar
     return REDISMODULE_OK;
 }
 
+static int parseAlterArgs(RedisModuleCtx *ctx, RedisModuleString **argv, int argc,
```

Comment:
@otnielvh why create another function instead of using `parseCreateArgs`?

Reply:
@danni-m, we agreed TS.ALTER won't change the chunk size and therefore parseCreateArgs won't suffice. We can think of a more elegant way to parse the input, but that is beyond the scope of this PR